typesetting/aegisub.md

```diff
@@ -61,105 +61,115 @@ Here is a list of tools you will want to download:
+
+![Aegisub 8975-master-8d77da3 script properties 1](images/script_properties-1.png)
+
+For most cases with modern fansubbing,
+the BT.709 color space will be used
+as opposed to the legacy BT.601 color space.
+If you want a more in-depth explanation of color matrices
+and how these two are different,
+you can read up [here](../archived-websites/bt601-vs-bt709.md),
+but the gist of it is this:
+BT.601 is for Standard Definition video
+and BT.709 is for High Definition video[^3].
+
+Manually setting the script to BT.601 could
+*irreversibly ruin the colors of any typesetting,
+dialogue,
+or kfx already in the script*.
+Even worse,
+some video renderers will read this setting from the muxed subtitles
+and render the video to match it.
+
+
+If you are working on a DVD
+or something in Standard Definition,
+you can change this to BT.601 manually in *File \> Script Properties*.
+However, not all Standard Definition video will be BT.601,
+so when in doubt,
+ask the encoder or check the source's
+[MediaInfo][] if they are not available.
 
-[bt601-vs-bt709]: http://blog.maxofs2d.net/post/148346073513/bt601-vs-bt709
 [MediaInfo]: https://mediaarea.net/en/MediaInfo
 
-The "Subtitles Provider".
-
-1.  Just a few years ago, there was a pretty clear consensus on which
-    subtitle renderer to use for anime and softsubs. These days, not so
-    much.
-2.  It used to be that
-    [VSFilter][] was
-    the best renderer around and was the only supported renderer by most
-    fansub groups.
-3.  However, it was eventually replaced
-    with [xy-vsfilter][] and
-    then that was replaced
-    with [xySubFilter][]
-    because VSFilter and xy-vs filter were not keeping up with the
-    demands of subtitles.
-4.  By 2015, however, xySubFilter development had come to a halt and
-    since then, [libass][] has made many
-    improvements both in speed and compatibility with advanced
-    subtitling in part due to contributions from members of the fansub
-    community.
-5.  At the end of the day, which renderer you choose is up to you, but
-    we recommend libass. It is maintained, cross-platform, able to handle
-    most typesetting, and has been integrated into many commercial and
-    open source software products. If you prefer, however, stock Aegisub
-    ships with VSFIlter and line0's builds come with xy-vsfilter.
+
+### The "Subtitles Provider"
+
+The recommended build of Aegisub comes pre-equipped with libass,
+so no manual settings change is needed.
+The following is a brief history of subtitle renderers.
+
+Just a few years ago,
+there was a pretty clear consensus on which
+subtitle renderer to use for anime and softsubs.
+These days, not so much.
+It used to be that [VSFilter][] was the best renderer around
+and was the only supported renderer by most fansub groups.
+However, it was eventually replaced with [xy-vsfilter][]
+and then later replaced with [xySubFilter][]
+because VSFilter and xy-vsfilter were not keeping up
+with the demands of subtitles.
```

Comment:
Probably noteworthy that it's about the *resource requirements* of subtitles or rather that the performance of renderers could not keep up. Functionally they are still the same.